config_task_limits.go

```diff
@@ -41,6 +41,9 @@ type TaskLimitsConfig struct {
 
 	// MaxParserProjectSize is the maximum allowed parser project size.
 	MaxParserProjectSize int `bson:"max_parser_project_size" json:"max_parser_project_size" yaml:"max_parser_project_size"`
+
+	// MaxTaskSecs is the maximum number of seconds a task can run and set their timeout to.
+	MaxTaskSecs int `bson:"max_task_secs" json:"max_task_secs" yaml:"max_task_secs"`
```

Comment:
nit: there's multiple timeouts in the agent that control subtly different things, can we call it something more specific, like `MaxExecTimeoutSecs`?

Reply:
done